web/src/components/apps/ClusterNodes.jsx

```diff
@@ -64,6 +64,10 @@ export class ClusterNodes extends Component {
       const response = await res.json();
       this.setState({
         kurl: response,
+        // if cluster doesn't support ha, then primary will be disabled. Force into secondary
+        selectedNodeType: !response.ha
+          ? "secondary"
+          : this.state.selectedNodeType,
```

Comment:
`selectedNodeType: primary` is not supported if `kurl.ha = false` so force the selection to secondary when this is the case. Otherwise leave the value as is. 

https://github.com/replicatedhq/kots/blob/9a6623cbc35da2cae4c52b72f5a4a2a0359576a4/web/src/components/apps/ClusterNodes.jsx#L326-L362